spec/ransack/adapters/active_record/base_spec.rb

```diff
@@ -145,6 +145,9 @@ def self.sane_adapter?
           end
 
           it "should function correctly with a multi-parameter attribute" do
+            ::ActiveRecord::Base.default_timezone = :utc
```

Comment:
Would have been better to have this in `config/environments/test.rb`, as this could lead to unexpected behaviour for other specs.